tests/fast-integration/test/fixtures/commerce-mock/application.yaml

```diff
@@ -3,7 +3,53 @@ kind: Application
 metadata:
   name: commerce
 spec:
-  labels: 
-    override.deployment.args.skipVerify: "true"
+  accessLabel: commerce
   description: Commerce mock
+  labels:
+    override.deployment.args.skipVerify: "true"
+  services:
+  - description: These services manage all of the common commerce functionality, and
+      also include customizations from installed AddOns. The implementing extension
+      is called ycommercewebservices.
+    displayName: SAP Commerce Cloud - Commerce Webservices
+    entries:
+    - accessLabel: commerce-468ed067-dacb-4009-a9d5-497e3d2333c8
+      centralGatewayUrl: http://central-application-gateway.kyma-system:8080/commerce/sap-commerce-cloud-commerce-webservices
+      credentials:
+        authenticationUrl: http://commerce-mock.mocks.svc.cluster.local:10000/rest/v2/authorizationserver/oauth/token
+        secretName: commerce-468ed067-dacb-4009-a9d5-497e3d2333c8
+        type: OAuth
+      targetUrl: http://commerce-mock.mocks.svc.cluster.local:10000/rest/v2
+      type: API
+    id: 468ed067-dacb-4009-a9d5-497e3d2333c8
+    labels:
+      connected-app: commerce
+    longDescription: These services manage all of the common commerce functionality,
+      and also include customizations from installed AddOns. The implementing extension
+      is called ycommercewebservices.
```

Comment:
Should we adjust the description somehow? Maybe it's no longer that wide spectrum of functionalities anymore